User Guide 3.2/Tests/test-scoring-and-outcome-declarations.md

```diff
@@ -0,0 +1,50 @@
+<!--
+tags: []
+
+--> 
+
+# Test Scoring Rules and Outcome Declarations
+
+> Test scores are determined by a student's performance in the various [Items](../appendix/glossary.md#item) of a [Test](../appendix/glossary.md#test). Individual item scores can be tallied using different methods to produce the final Test result. This chapter shows how to configure the Test scoring rules, and compute the Outcome Declarations.
+
+>For information on Item Scoring Rules, see [Item Scoring Rules](../items/item-scoring-rules.md).
+
+
+Follow the steps below to set up your chosen scoring method.
+
+**1.** After you have created your Test by selecting the Items which are to go in it, hover over the cog wheels just below the Action Bar, at the top of the right-hand corner of the canvas, and you will see the option *Manage Test Properties*. Click on this and the properties relating to the scoring of Tests will open in the right-hand panel. 
+
+
+**2.** Click on the button *Scoring*.
+
+There are four potential configurations from which to choose how you would like the Test to be scored. Enter your choice in *Outcome Processing*:
+
+<!-- Missing Screenshot: Scoring -->
+
+- **None**: If this option is selected, any existing scoring rules will be removed. In this case, no test score will be generated. Scores for the individual items of the Test can be extracted by clicking on *Results*, or the QTI Results API. Use this option if you want to do full custom processing.
+
+- **Custom**: This option is only applicable when a test with custom outcome processing rules is imported - thus the processing rules are defined outside of TAO. In this case, the rules cannot be authored by TAO, so the existing rules are left untouched.
```

Comment:
This would be the case if the assessmentTest XML would be modified outside of TAO and imported back into the system.